packages/next/src/build/index.ts

```diff
@@ -3085,48 +3135,24 @@ export default async function build(
           })
       }
 
-      if (config.output === 'export') {
-        if (buildTracesSpinner) {
-          buildTracesSpinner?.stop()
-          buildTracesSpinner = undefined
-        }
+      await buildTracesPromise
 
-        const exportApp: typeof import('../export').default =
-          require('../export').default
+      if (buildTracesSpinner) {
```

Comment:
this spinner should end after `config.output === 'export'`?

Reply:
As discussed I found that the spinner was wrong 👍